plugins/modules/device_credential_workflow_manager.py

```diff
@@ -2513,8 +2504,7 @@ def verify_diff_merged(self, config):
             self.log("Successfully validated the assign device credential to site", "INFO")
             self.result.get("response")[0].get("assignCredential").update({"Validation": "Success"})
 
-        self.msg = "Successfully validated the Global Device Credential and \
-                    Assign Device Credential to Site."
+        self.msg = "Successfully validated the Global Device Credential and Assign Device Credential to Site."
```

Comment:
Successfully validated the global device credential and **assigned** the device credential to the site.